katello/katello-change-hostname

```diff
@@ -0,0 +1,190 @@
+             "All services will be stopped and started during this process. This script will modify your system!")
+STDOUT.print("\nHave you made this necessary change and want to proceed with changing your hostname to #{new_hostname}? [y/n]")
+response = yesno
+STDOUT.puts ""
+
+unless response
+  STDOUT.puts "Hostname change aborted, no changes have been made to your system"
+  exit
+end
+
+# Get the hostname from your system
+old_hostname = get_hostname
+
+if is_rhel_version?("6")
+  STDOUT.puts "updating hostname in /etc/sysconfig/network"
+  STDOUT.puts("sed -i -e 's/#{old_hostname}/#{new_hostname}/g' /etc/sysconfig/network")
+  `sed -i -e 's/#{old_hostname}/#{new_hostname}/g' /etc/sysconfig/network`
+  STDOUT.puts "setting hostname"
+  `hostname #{new_hostname}`
+elsif is_rhel_version?("7")
+  STDOUT.puts "updating hostname in /etc/hostname"
+  `sed -i -e 's/#{old_hostname}/#{new_hostname}/g' /etc/hostname`
+  STDOUT.puts "setting hostname"
+  `hostnamectl set-hostname #{new_hostname}`
+else
+  STDOUT.puts "couldn't find RHEL version: #{RHEL_VERSION} - exiting, no changes have been made to your system"
+  exit
+end
+
+STDOUT.puts "checking if hostname was changed"
+if get_hostname != new_hostname
+  STDOUT.puts "the new hostname was not changed successfully, exiting script"
+  exit
+end
+
+STDOUT.puts "stopping services"
+`katello-service stop`
+
+STDOUT.puts "deleting old certs"
+`
+rm -rf /etc/pki/katello{,.bak}
+rm -rf /etc/pki/katello-certs-tools{,.bak}
+rm -rf /etc/candlepin/certs/amqp{,.bak}
+rm -rf /etc/foreman-proxy/*ssl*
+rm -rf /etc/foreman/old-certs
+rm -rf /etc/foreman/*.pem
+rm -rf /var/lib/puppet/ssl
+rm -rf /etc/pki/katello/nssdb
+rm -f /etc/tomcat/keystore
+rm -f /etc/pki/katello/keystore
+cp -r #{options[:ssl_dir]} /root/ssl-build.#{old_hostname}#{timestamp}
+rm -rf /root/ssl-build
+`
+STDOUT.puts "backed up #{options[:ssl_dir]} to /root/ssl-build.#{old_hostname}#{timestamp}"
+STDOUT.puts "updating hostname in /mtc/hosts"
+`sed -i -e 's/#{old_hostname}/#{new_hostname}/g' /etc/hosts`
+
+STDOUT.puts "updating hostname in foreman installer scenarios"
+`sed -i -e 's/#{old_hostname}/#{new_hostname}/g' /etc/foreman-installer/scenarios.d/*.yaml`
+
+STDOUT.puts "removing last_scenario.yml file"
+`rm -rf /etc/foreman-installer/scenarios.d/last_scenario.yaml`
+
+STDOUT.puts "re-running the installer"
+
+installer = "#{options[:program]}-installer --scenario #{options[:scenario]} -v --certs-regenerate-ca=true --certs-regenerate=true \
+  --foreman-foreman-url https://#{new_hostname} \
+  --foreman-servername #{new_hostname} \
+  --capsule-qpid-router-broker-addr  #{new_hostname}\
+  --certs-ca-common-name #{new_hostname} \
+  --certs-node-fqdn #{new_hostname} \
+  --capsule-parent-fqdn #{new_hostname}\
+  --foreman-proxy-register-in-foreman true"
+
+installer << " --disable-system-checks" if options[:system_check]
+STDOUT.puts installer
+installer_output = `#{installer}`
+STDOUT.puts installer_output
```

Comment:
Does this mean I won't see the installer output in real time? If not, as a user, what do I see when this is running to know my progress?